include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -1007,6 +1007,7 @@ Row |       warp 0                warp 2
   let parameters = (
     ins
     "unsigned": $version,
+    "bool":$isTransposed,
```

Comment:
Please add documentation for this new field.

Reply:
added here: https://github.com/triton-lang/triton/pull/5456/files#diff-b359f0617893941d9de173665ce05fb4b3e8bba11a7750c4d55df1d86d674650R1005